recipe-server/normandy/settings.py

```diff
@@ -328,6 +329,11 @@ def RAVEN_CONFIG(self):
 
     PROD_DETAILS_DIR = values.Value(os.path.join(Core.BASE_DIR, 'product_details'))
 
+    # AWS settings
+    AWS_ACCESS_KEY_ID = values.Value()
+    AWS_SECRET_ACCESS_KEY = values.Value()
+    AWS_STORAGE_BUCKET_NAME = values.Value()
```

Comment:
sure, we can try that. although i had trouble locally where it did not read those variables from the environment and i had to populate these settings. but i might have been doing something wrong.